test/testrecipes/test-terraform-recipes/postgres/main.tf

```diff
@@ -69,13 +69,13 @@ resource "kubernetes_service" "postgres" {
   }
 }
 
-resource "time_sleep" "wait_20_seconds" {
+resource "time_sleep" "wait_50_seconds" {
   depends_on = [kubernetes_service.postgres]
-  create_duration = "20s"
+  create_duration = "50s"
```

Comment:
Why 50 and not 60?

Reply:
it's an arbitrary number closer to double the current timeout. I'm okay to check if tests work with this value